src/handlers/logging.js

```diff
@@ -0,0 +1,5 @@
+import { configureLoggingService } from '@edx/frontend-logging';
+
+export default async function logging(app) {
+  configureLoggingService(app.loggingService);
```

Comment:
I don't quite understand what's happening here, but I know it's not new. We have a logging service and then configuring our own with it?

Reply:
We're choosing a logging service implementation for the app.  It's basically what makes it so you can import "logError" from frontend-logging without needing to import NewRelicLoggingService instead and call its logError method.